x/proof/keeper/msg_server_submit_proof_test.go

```diff
@@ -270,9 +286,11 @@ func TestMsgServer_SubmitProof_Error_OutsideOfWindow(t *testing.T) {
 
 	// Advance the block height to the claim window open height.
 	sharedParams := keepers.SharedKeeper.GetParams(ctx)
-	claimMsgHeight := shared.GetClaimWindowOpenHeight(
+	claimMsgHeight := shared.GetEarliestClaimCommitHeight(
 		&sharedParams,
 		sessionHeader.GetSessionEndBlockHeight(),
+		claimWindowOpenHeightBlockHash,
```

Comment:
Where is this set?

Reply:
It's not, it can be any value for this test, including the byte slice zero value .